### PR TITLE
Tests tool tip modal

### PR DESCRIPTION
**Pull request by sjpaige**
Wrote tests for the ToolTipModal Component.

### CODE REVIEW

File: src/Components/__tests__/ToolTipModal.spec.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import ToolTipModal from '../ToolTipModal';
+
+describe("<ToolTipModal />", () => {
+    let wrapper;
+    beforeEach(() => {
+        wrapper = mount(<ToolTipModal text={"test"} />)
+    })
+
+    it("renders a modal", () => {
+        expect(wrapper.find(Modal)).toHaveLength(1);
```
Comment (stuttskl) [inline src/Components/__tests__/ToolTipModal.spec.js:16]: If we're just testing if there is 1 node (i.e. if something exists or not) how do you feel about using the[ `exists` selector? ](https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/exists.html)

File: src/Components/__tests__/ToolTipModal.spec.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    it("renders a button", () => {
+        expect(wrapper.find(Button)).toHaveLength(1);
+    });
+
+    it("displays passed in props tooltip data on the button", () => {
+        expect(wrapper.find(Button).text()).toEqual("test");
+    })
+
+    it("simulates button opening and closing modal", () => {
```
Comment (stuttskl) [inline src/Components/__tests__/ToolTipModal.spec.js:27]: Let's  separate our tests by "category", for lack of a better word. For example, we can separate UI tests and behavior/functionality tests like so:
```
describe("ui", () => {
    it("should render a modal", () => {
        ...
    })

    it("should render a button", () => {
        ...
    })

    it("should render the correct button text", () => {
        ...
    })
})

describe("clicking on the button", () => {
    it("should close the modal", () => {
        ...
    })

    it("should open the modal", () => {
        ...
    })
})
```

The main idea here being: organizing our tests under specific `describe` statements, and having our tests read like English sentences in the output. "clicking on the button...should close the modal" etc.

File: src/Components/__tests__/ToolTipModal.spec.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        expect(wrapper.find(Modal).props().open).toBe(false);
+        wrapper.find(Button).simulate('click');
+        expect(wrapper.find(Modal).props().open).toBe(true);
+        wrapper.find(Backdrop).simulate('click');
+        expect(wrapper.find(Modal).props().open).toBe(false);
+    })
+
```
Comment (stuttskl) [inline src/Components/__tests__/ToolTipModal.spec.js:34]: 3 extra newlines here

**Review by stuttskl — COMMENTED** [review]
Please see comments and lmk what you think :)

File: src/Components/__tests__/ToolTipModal.spec.js (hunk 1 above)
Comment (sjpaige) [reply src/Components/__tests__/ToolTipModal.spec.js:16]: That selector makes a lot more sense for those type of tests.

File: src/Components/__tests__/ToolTipModal.spec.js (hunk 2 above)
Comment (sjpaige) [reply src/Components/__tests__/ToolTipModal.spec.js:27]: That makes sense. I will update it to reflect these ideas.

**Comment by sjpaige** [comment]
> Please see comments and lmk what you think :)

Great ideas! I went ahead and made the changes you suggested.

**Review by stuttskl — APPROVED** [review]
Nice! This looks great =)

**Pull request merged**